soc/arm/nordic_nrf/Kconfig.peripherals

```diff
@@ -133,9 +133,6 @@ config HAS_HW_NRF_RADIO_BLE_CODED
 config HAS_HW_NRF_RADIO_IEEE802154
```

Comment:
This Kconfig option should be replaced by an entry from DT indicating if given SoC supports IEEE 802.15.4 protocol